cypress/integration/settingsPage/updateUser/updateUser.ts

```diff
@@ -0,0 +1,121 @@
+import SharedDataUtils from "@pageObjects/dataUtils";
+import ProfileAssertion from "@pageObjects/profile/assertions";
+import SettingsPageActions from "@pageObjects/settings/actions";
+import SettingsPageAssertions from "@pageObjects/settings/assertions";
+import { NewUser, NewUserResponseBody } from "@support/types";
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+import moment from "moment";
+
+const settingsPageActions = new SettingsPageActions();
+const settingsPageAssertions = new SettingsPageAssertions();
+const profileAssertion = new ProfileAssertion();
+const sharedDataUtils = new SharedDataUtils();
+
+const user: NewUser = {
+  username: "ConduitUser",
+  email: "conduituser@email.com",
+  password: "123456",
+};
+
+const newUser: NewUserResponseBody = {
```

Comment:
update this one please, use the NewUser type instead of the NewUserResponseBody, you need to update the NewUser type by adding two new values 'bio' and 'image', and make them optional.